src/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 # Allow the source files to find headers in src/
 include_directories(${PROJECT_SOURCE_DIR}/src)
+# Allow the source files to find headers in include/
```

Comment:
BENCHMARK_MAIN() is currently defined in publicly exposed interface (include/ folder), and it is now required in src/ folder.
Unfortunately, there is no common place that is shared between src/ and include/: they are decoupled. Otherwise, I would move BENCHMARK_MAIN() there.
I added comment explaining why it's now needed.